source/puppet/4.4/reference/install_osx.md

```diff
@@ -29,94 +38,119 @@ To install on other operating systems, see the pages linked in the navigation si
 {% include puppet-collections/_puppet_collection_1_osx1010.md %}
 {% include puppet-collections/_puppet_collection_1_osx1009.md %}
 
-You can also download [older versions of Puppet](https://downloads.puppetlabs.com/mac/); browse to `<OS X VERSION>/PC1/x86_64` for the most recently released packages.
-
-These packages are tied to Puppet Collection 1, which is a set of Puppet software designed to work well with Puppet 4. The `puppet-agent` package bundles all of Puppet's prerequisites, so you don't need to download anything else to install Puppet on an agent node.
+You can also download older versions of Puppet from [downloads.puppetlabs.com](https://downloads.puppetlabs.com/mac/). Browse to `<OS X VERSION>/PC1/x86_64` for the most recently released packages. For example, the path to the PC1 package for OS X 10.11 is `https://downloads.puppetlabs.com/mac/10.11/PC1/x86_64`.
 
 ### Choosing a package
 
-OS X packages are named according to their `puppet-agent` version and compatible OS X version:
+Packages are named according to their `puppet-agent` version and compatible OS X version. They also contain a release number --- typically `1` --- that represents the iteration of that version's package.
 
-    puppet-agent-<PACKAGE VERSION>.osx<OS X VERSION>.dmg
+    puppet-agent-<PACKAGE VERSION>-<RELEASE NUMBER>.osx<OS X VERSION>.dmg
```

Comment:
Per @melissa, all OS X packages now append a release number to the version number. This note needs to be backported to all Puppet docs relevant to `puppet-agent` since 1.2.4.